test/integration/features/cert_rotation.feature

```diff
@@ -0,0 +1,25 @@
+@cert_rotation @linux
+Feature: Check the cert is rotation happen after it expire
+
+    The user try to use crc after one month. he/she expect the
+    cert rotation happen successfully and able to deploy the app and check its
+    accessibility.
+
+    Scenario: Set clock to 3 month ahead on the host
+      Given executing "sudo systemctl stop chronyd" succeeds
+      Then executing "sudo date -s '3 month'" succeeds
```

Comment:
Maybe this should also be done by `crc --stop-chronyd`?

Reply:
@cfergeau that would be platform depend so I would prefer to have user decide on this.

Reply:
Feels odd to add code for half of the solution to crc, and declare the user is on their own for the other half.